taichi/codegen/llvm/codegen_llvm.cpp

```diff
@@ -2608,6 +2633,38 @@ llvm::Type *TaskCodeGenLLVM::get_mesh_xlogue_function_type() {
                                  get_mesh_xlogue_argument_types(), false);
 }
 
+llvm::PointerType *TaskCodeGenLLVM::get_integer_ptr_type(int bits) {
+  switch (bits) {
+    case 8:
+      return llvm::Type::getInt8PtrTy(*llvm_context);
+    case 16:
+      return llvm::Type::getInt16PtrTy(*llvm_context);
+    case 32:
+      return llvm::Type::getInt32PtrTy(*llvm_context);
+    case 64:
+      return llvm::Type::getInt64PtrTy(*llvm_context);
+    default:
+      break;
+  }
+  return nullptr;
+}
+
+llvm::IntegerType *TaskCodeGenLLVM::get_integer_type(int bits) {
+  switch (bits) {
+    case 8:
+      return llvm::Type::getInt8Ty(*llvm_context);
+    case 16:
+      return llvm::Type::getInt16Ty(*llvm_context);
+    case 32:
+      return llvm::Type::getInt32Ty(*llvm_context);
+    case 64:
+      return llvm::Type::getInt64Ty(*llvm_context);
+    default:
+      break;
```

Comment:
We should raise an error here instead of returning nullptr. You can use the macro `TI_ERROR` to raise the error.